pact/verify.py

```diff
@@ -125,6 +144,9 @@ def main(pacts, base_url, pact_url, pact_urls, states_url,
         '--provider-base-url': base_url,
         '--provider-states-setup-url': states_setup_url,
         '--broker-username': username,
+        '--pact-broker-base-url': broker_base_url,
+        '--provider': provider,
+        '--consumer-version-tag': consumer_version_tag,
```

Comment:
When the python interface has been updated to allow multiple tags,  `--consumer-version-tag` can be called once for each tag. Ditto for `--provider-version-tag`.